java/java-ranger-regression/printtokens_eqchk/impl/token_stream.java

```diff
@@ -7,26 +7,26 @@
 public class token_stream {
```

Comment:
Preferrred (usual Java coding conventions)
```suggestion
public class TokenStream {
```

Reply:
While I see your point here, it seems like the Java ranger code base has a couple of test cases diverging with this convention. As this is also valid Java, I would vote to allow underscore names in the benchmark tasks. This makes it easier to contribute test cases for them. Otherwise #1154 requires a lot of refactoring as well.